spanner/integration_test.go

```diff
@@ -3697,6 +3783,15 @@ func skipEmulatorTest(t *testing.T) {
 	}
 }
 
+func skipUnsupportedPGTest(t *testing.T) {
+	if testDialect == adminpb.DatabaseDialect_POSTGRESQL {
+		skipEmulatorTest(t)
+		if _, ok := supportedPGTests[t.Name()]; !ok {
```

Comment:
`supportedPGTests` seems not to be necessary. If we don't want to skip a test, we don't call `skipUnsupportedPGTest`. E.g., 

```
func skipEmulatorTestForPG(t *testing.T) {
	if isEmulatorEnvSet() && testDialect == adminpb.DatabaseDialect_POSTGRESQL {
		t.Skip("Skipping PG testing against the emulator.")
	}
}

func skipUnsupportedPGTest(t *testing.T) {
	if testDialect == adminpb.DatabaseDialect_POSTGRESQL {
		t.Skip("Skipping testing of unsupported tests in Postgres dialect.")
        }
}

// Only run against the production, not the emulator
func TestIntegration_SingleUse(t *testing.T) {
        skipEmulatorTestForPG(t)
        ...
}

// Do not run for PG
func TestIntegration_DirectPathFallback(t *testing.T) {
        skipUnsupportedPGTest(t)
        ...
}
```